commands/utilities/1984.js

```diff
@@ -0,0 +1,38 @@
+const { SlashCommandBuilder, Message } = require("discord.js");
+const { blacklist } = require('../../utility/blacklist');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("1984")
+        .setDescription("Deletes a specified number of messages from a specific user within 1000 latest messages.")
+        .addStringOption(option =>
+            option.setName('user_id')
+            .setDescription('ID of user whose messages you want to delete')
+            .setRequired(true)
```

Comment:
I noticed you have setRequired(true) here, but not in the other functions, can they be added in the previous comments?